cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
   private checkout: string;
 
   constructor (){
-    this.addToCartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkout = "[style*='display: block;'] .button-container > a";
+    this.addToCartButton = ".ajax_add_to_cart_button > span";
+    this.checkout = ".button-container > .button-medium > span";
```

Comment:
here you can use the text of the button and then a contains in the method that this selector is used, or, you can use an attribute to indicate you are referring to the Proceed to checkout button.